lms/static/scripts/frontend_apps/utils/api.ts

```diff
@@ -169,7 +173,7 @@ export function urlPath(strings: TemplateStringsArray, ...params: string[]) {
  */
 export function useAPIFetch<T = unknown>(
   path: string | null,
-  params?: Record<string, string>,
+  { params, ...fetchOptions }: UseAPIFetchOptions = {},
```

Comment:
I considered adding a third param with the options, and refactor it later. However, I have seen we were not using `params` at all anywhere, so I decided to finally merge that inside the options object itself.